diff --git a/koboplatformplugin.pro b/koboplatformplugin.pro
index 7a40246..e817041 100644
--- a/koboplatformplugin.pro
+++ b/koboplatformplugin.pro
@@ -21,7 +21,7 @@ QT +=  widgets \
 INCLUDEPATH += $$PWD/src
 
 # FBINK
-DEFINES += FBINK_FOR_KOBO
+DEFINES += FBINK_FOR_KINDLE
 FBInkBuildEvent.input = $$PWD/FBink/*.c $$PWD/FBink/*.h
 PHONY_DEPS = .
 FBInkBuildEvent.input = PHONY_DEPS
@@ -36,12 +36,12 @@ INCLUDEPATH += $$PWD/FBInk $$PWD/FBInk/libi2c-staged/include
 LIBS += -L$$PWD/FBInk/libi2c-staged/lib/ -l:libi2c.a
 
 CONFIG(debug, debug|release) {
-    FBInkBuildEvent.commands = CROSS_TC=$$CROSS_TC MINIMAL=1 DRAW=1 DEBUG=1 KOBO=true make -C $$PWD/FBInk pic
+    FBInkBuildEvent.commands = CROSS_TC=$$CROSS_TC MINIMAL=1 DRAW=1 DEBUG=1 KINDLE=true make -C $$PWD/FBInk pic
     LIBS += -L$$PWD/FBInk/Debug -l:libfbink.a
 }
 
 CONFIG(release, debug|release) {
-FBInkBuildEvent.commands = CROSS_TC=$$CROSS_TC MINIMAL=1 DRAW=1 KOBO=true make -C $$PWD/FBInk pic
+FBInkBuildEvent.commands = CROSS_TC=$$CROSS_TC MINIMAL=1 DRAW=1 KINDLE=true make -C $$PWD/FBInk pic
     LIBS += -L$$PWD/FBInk/Release -l:libfbink.a
 }
 
diff --git a/src/kobodevicedescriptor.h b/src/kobodevicedescriptor.h
index 6e3008d..e99bace 100644
--- a/src/kobodevicedescriptor.h
+++ b/src/kobodevicedescriptor.h
@@ -64,7 +64,7 @@ struct KoboDeviceDescriptor
     // Stable path to the NTX input device
     QString ntxDev = "/dev/input/event0";
     // Stable path to the Touch input device
-    QString touchDev = "/dev/input/event1";
+    QString touchDev = "/dev/input/event2";
 };
 
 KoboDeviceDescriptor determineDevice();
